ansible/roles/software/ckan/catalog/ckan-app/templates/catalog-next/etc_ckan_production.ini.j2

```diff
@@ -188,7 +188,9 @@ ckan.hide_activity_from_users = %(ckan.site_id)s
 #smtp.user = your_username@gmail.com
 #smtp.password = your_password
 smtp.mail_from = {{ catalog_ckan_email_from }}
-
+{% if ckan_email_enabled|default(False) %}
```

Comment:
Sorry, I was not reading this closely when I made my suggestion.

1. If you set the variable in the [role defaults](https://github.com/GSA/datagov-deploy/blob/develop/ansible/roles/software/ckan/catalog/ckan-app/defaults/main.yml), you don't need `default(False)`, which makes it less error prone if someone uses the variable elsewhere.
2. since this is enabling the test server and essentially _disabling_ email, let's call it `ckan_email_test_server_enabled` instead.
3. I wasn't reading the CKAN code closely, I thought it was just looking for a truthy value, but looks like it will actually try to open a connection? What's the right value to put in `smtp.test_server`?

Reply:
@adborden I added [this changes](https://github.com/GSA/datagov-deploy/pull/1953/commits/69b6aae12031587e7ad320559c401624a5a91f79)
The test server will also [start a connection](https://github.com/GSA/ckan/blob/datagov-newcatalog/ckan/lib/mailer.py#L56-L68) but without TLS, user and password.

Reply:
Hmm, so if CKAN still opens a connection and we have`smtp.test_server = localhost`, won't it still connect to the live postfix service on localhost:25?